avro-patches.gemspec

```diff
@@ -24,7 +24,6 @@ Gem::Specification.new do |spec|
   end
 
   spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
-  spec.bindir        = 'bin'
   spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
```

Comment:
Should these files be excluded from the gem entirely? Is there any reason to distribute them if they are only intended for development?

Reply:
Since it's only intended for development, probably removing them from the gem makes sense